src/snakemake/runtime_dependencies.py

```
@@ -0,0 +1,270 @@
+            return
+
+        prior_packages = {}
+        for pkg_set in prior_package_sets:
+            for pkg in pkg_set:
+                prior_packages[pkg.name] = pkg
+
+        # add the actually requested packages (may overwrite versions)
+        for pkg in self._packages[package_type].values():
+            prior_packages[pkg.name] = pkg
+
+        breakpoint()
```

Ruff (0.15.18)

[error] Trace found: `breakpoint` used

(T100)